CHANGELOG.md

```diff
@@ -74,6 +74,7 @@
 - Make ExecutionSpace::concurrency() a non-static member function [\#5655](https://github.com/kokkos/kokkos/pull/5655) and related PRs
 
 ### Deprecations
+- Remove code guarded by `KOKKOS_ENABLE_DEPRECATED_CODE_3`
```

Comment:
I would rather list this under "Incompatibilities".

Reply:
done